buildSrc/src/main/java/org/elasticsearch/gradle/testfixtures/TestFixturesPlugin.java

```diff
@@ -45,21 +46,18 @@ public void apply(Project project) {
             "testFixtures", TestFixtureExtension.class, project
         );
 
-        // Don't look for docker-compose on the PATH yet that would pick up on Windows as well
```

Comment:
I see. We can't bail here because we won't do the disabling that we need. Got it!